aws/iam/audit.tf

```diff
@@ -6,7 +6,7 @@ variable "audit_account_user_names" {
 
 # Provision group access to audit account. Careful! Very few people, if any should have access to this account.
 module "organization_access_group_audit" {
-  source            = "git::https://github.com/cloudposse/terraform-aws-organization-access-group.git?ref=tags/0.2.1"
+  source            = "git::https://github.com/cloudposse/terraform-aws-organization-access-group.git?ref=add-switchrole-url"
```

Comment:
https://github.com/cloudposse/terraform-aws-organization-access-group/pull/11 has been approved. Once merged we can cut a release and use the tag here.